src/docs/product/integrations/gitlab/index.mdx

```diff
@@ -127,7 +127,7 @@ A `<SHORT-ID>` may look something like 'BACKEND-C' in the image below.
 <Note>
 
 This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
-If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+If you’re interested in being an , you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
```

Comment:
oof. I will remove. Thanks.

Reply:
```suggestion
If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
```